src/index.js

```diff
@@ -219,5 +219,9 @@ AsyncIterator.fromEvent = (emitter, event, options) => {
  */
 AsyncIterator.Backing = AsyncIteratorBacking;
 
+AsyncIterator.fromArray = fromArray;
+AsyncIterator.fromStream = fromStream;
+AsyncIterator.fromEvent = fromEvent;
+
 // export the iterator class
 export default AsyncIterator;
```

Comment:
Please change this to a named export. I would prefer to have either a single default export or all named exports.